example/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Demonstrates how to use the tahiti plugin.
 dependencies:
   flutter:
     sdk: flutter
-  flutter_svg: ^0.6.3
+  flutter_svg: ^0.5.1
```

Comment:
Is this version downgrade needed?

Reply:
"Because every version of tahiti from path depends on flutter_svg ^0.6.3 and tahiti_example depends on flutter_svg ^0.5.1, tahiti from path is forbidden." 
like error I'm getting while using flutter_svg ^0.6.3 .

Reply:
pubspec.yaml updated No need of version downgrade.